src/controllers/countries.js

```
@@ -0,0 +1,32 @@
+const countriesService = require('../services/countries');
+
+const fetchCountries = async (req, res) => {
+  try {
+    const countries = await countriesService.getCountries();
+    res.json(countries);
+  } catch (error) {
+    console.error(error);
```

ESLint

[error] Extra semicolon.

(semi)